pom.xml

```diff
@@ -268,8 +268,8 @@ SOFTWARE.
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.0</version>
           <configuration>
-            <source>1.8</source>
-            <target>1.8</target>
+            <source>11</source>
```

Comment:
@paulodamaso What about moving the versions to 
`<properties>...</properties>`
section in pom.xml ?